src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -179,6 +189,16 @@ public void Close(bool exitHostProcess = true)
             if (exitHostProcess)
             {
                 CloseHostProcess();
+
+                AppDomain.CurrentDomain.FirstChanceException -= FirstChanceExceptionHandler;
+
+                if (!_exceptionHappened)
+                {
+                    if (Directory.Exists(DumpDirectoryPath))
```

Comment:
Don't think we should be doing this.  What if another part of our Jenkins infrastructure wrote out a dump file?  This code would then be deleting it.